cluster/gce/manifests/cluster-autoscaler.manifest

```diff
@@ -56,6 +56,11 @@
                         "readOnly": true,
                         "mountPath": "/usr/share/ca-certificates"
                     },
+                    {
+                        "name": "srvkube",
+                        "readOnly": true,
+                        "mountPath": "/etc/srv/kubernetes"
```

Comment:
only mount /etc/srv/kubernetes/cluster-autoscaler

Reply:
Changed this, using ```srv_kube_path``` variable now.